docs/getting_started/basics/execute.md

```diff
@@ -34,22 +36,65 @@ The "Execute Next" action will compute the next dirty node present in the workfl
 </figure>
 
 
-## ![Clear Execution Button](images/clear_all_default.svg){style="width:5%"} Clear Execution
+### ![Clear Execution Button](images/clear_all_default.svg){style="width:5%"} Clear Execution
 
-The "Clear Execution" action will set all the nodes in the workflow to dirty. All **connected** inputs and output values will be cleared to their default values. This forces Shift to recompute all nodes the next time the workflow has to be executed. 
+The "Clear Execution" action will set all the nodes in the workflow to dirty. All **connected** inputs and output values will be cleared to their default values. This forces Shift to recompute all nodes the next time the workflow has to be executed.
 
 <figure>
       <img src="images/clear_execution.gif" alt="Clear execution">
       <figcaption><b>Figure 4</b>: Clear the executed status from all nodes.</figcaption>
 </figure>
 
 
-## ![Live Execution Button](images/execute_live_default.svg){style="width:5%"} Live Execution
+### ![Live Execution Button](images/execute_live_default.svg){style="width:5%"} Live Execution
 
 The "Live Execution" action allows to trigger the execution of the whole workflow whenever a change to any node is done. While the "Live Execution" is turned on, the other execution buttons are disabled. To re-enable all the execution buttons, turn Live Execution off.
 
 <figure>
-      <img src="images/live_execution.gif" alt="Live Execution">
+      <img src="images/stopping_execution.gif" alt="Live Execution">
       <figcaption><b>Figure 5</b>: Executing workflow in live execution mode.</figcaption>
 </figure>
 
+## Threaded Execution
+
+By default Shift will run the execution of the workflow on a dedicated thread. This allows the user to freely inspect the board contextually to its execution. Additionally, for long executing workflows, the user is free to author and inspect any board other than the executing one.
```

Comment:
I don't understand what "inspect the board contextually to its execution" really means. Is there a way to reword it?

Reply:
I rephrased it